plotly/plotly/plotly.py

```diff
@@ -1449,6 +1454,194 @@ def _send_to_plotly(figure, **plot_options):
     return r
 
 
+def get_grid(grid_url, raw=False):
+    """
+    Returns a JSON figure representation for the specified grid.
```

Comment:
🎎 This feels a little misleading. Aren't we returning either JSON or a custom object. And, in the JSON case, it's a dict, right.